### ISSUE TITLE
Live Editable Calendar and saving the changes to a file

### ISSUE THREAD

**Issue by ekianjo** [dreamRs/toastui#23]
Hi, thanks a lot for this very useful package - 
I'm trying to see if it's possible to edit the calendar in a shiny app to add entries directly by clicking on a date, and it works front-end wise with: 

calendar(cal_demo_data(), navigation = TRUE,useCreationPopup = T,isReadOnly = F)

but after entering a new event the calendar does not reflect the changes. Is there a way to "update" the calendar data live after doing a new entry? If yes, could you add such an example in the reference? 

Thanks!

**Comment by pvictor** [comment]
Yes that's possible, updating the calendar needs to be done server-side, here's an example: https://github.com/dreamRs/toastui/blob/master/inst/examples/calendar-edit-schedules.R

Victor
